<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeliveryDebugWidget</class>
 <widget class="QWidget" name="DeliveryDebugWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>1054</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="TryToggle" name="tryToggleBirth" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonScrape">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Scrape the woom, resetting ALL identity (DANGEROUS!!)</string>
       </property>
       <property name="text">
        <string>Scrape</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PortableIDWidget" name="widgetBirthCertificate" native="true"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1,0,1">
     <item row="4" column="4">
      <widget class="LightWidget" name="widgetLightHasNodeIdentityGender" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="LightWidget" name="widgetLightHasNodeIdentityID" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="labelHasNodeIdentityRole">
       <property name="text">
        <string>Has Role</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="labelHasNodeIdentityID">
       <property name="text">
        <string>Has ID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="labelHasNodeIdentity">
       <property name="text">
        <string>Present</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="LightWidget" name="widgetLightHasNodeIdentityRole" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="LightWidget" name="widgetLightHasNodeIdentityName" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="labelHasNodeIdentityGender">
       <property name="text">
        <string>Has Gender</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="labelHasNodeIdentityName">
       <property name="text">
        <string>Has Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="LightWidget" name="widgetLightHasNodeIdentity" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="LightWidget" name="widgetLightHasNodeIdentityType" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="labelHasNodeIdentityType">
       <property name="text">
        <string>Has Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="LightWidget" name="widgetLightHasLocalKey" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="labelHasLocalKeyPublic">
       <property name="text">
        <string>Has public part</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="labelHasLocalKeyPrivate">
       <property name="text">
        <string>Has private part</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="LightWidget" name="widgetLightHasLocalKeyPrivate" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="LightWidget" name="widgetLightHasLocalKeyPublic" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="LightWidget" name="widgetLightHasLocalKeyID" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="labelHasLocalKeyID">
       <property name="text">
        <string>Has ID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="labelHasLocalKey">
       <property name="text">
        <string>Present</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="labelHasNodeIdentityIDMatchesLocalKeyID">
       <property name="text">
        <string>ID matches identity ID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="LightWidget" name="widgetLightHasNodeIdentityIDMatchesLocalKeyID" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="labelHasOKStatus_2">
       <property name="text">
        <string>Key</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLabel" name="labelHasOKStatus_3">
       <property name="text">
        <string>Identity</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="6">
      <widget class="QLabel" name="labelHasOKStatus">
       <property name="text">
        <string>All is OK</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonReload">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reload</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/update.svg</normaloff>:/icons/update.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="LightWidget" name="widgetLightAllOK" native="true">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LightWidget</class>
   <extends>QWidget</extends>
   <header location="global">components/LightWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TryToggle</class>
   <extends>QWidget</extends>
   <header location="global">components/TryToggle.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PortableIDWidget</class>
   <extends>QWidget</extends>
   <header location="global">node/PortableIDWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../libres/resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonScrape</sender>
   <signal>clicked()</signal>
   <receiver>DeliveryDebugWidget</receiver>
   <slot>scrapeWoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonReload</sender>
   <signal>clicked()</signal>
   <receiver>DeliveryDebugWidget</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>692</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scrapeWoom()</slot>
  <slot>reload()</slot>
 </slots>
</ui>
